app/components/container-subpod/template.hbs

```diff
@@ -1,10 +1,17 @@
-{{action-menu model=model showPrimary=false}}
 
 {{#if groupHasChildren}}
 <div class="subpod-children clearfix">
   <h6>Sidekicks</h6>
   {{#each children as |child|}}
-    {{container-dot model=child}}
+    {{#tooltip-element model=child}}
+      <i class="{{child.stateIcon}} {{child.stateColor}} dot hand"></i>
+      <span class="hide" id="tooltip-content">
```

Comment:
Maybe it's ok because there's only one in the DOM at a time, but `id="tooltip-content"` seems dangerous.  Can this be a class instead?  And can it have `hide` by default so you don't have to put it on every tooltip?